### ISSUE TITLE
Modify loss function per older yolo implementation

### ISSUE THREAD

**Issue by jpsember** [jpsember/java-ml#44]

**Comment by jpsember** [comment]
NaN appearing in tensors:
```
{"family_id": 0, "family_size": 0, "family_slot": 0, "id": 2, "message": "pred_wh", "shape": [3, 5, 2], "special_handling": 0, "tensor_floats": [NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN]}
```

**Comment by jpsember** [comment]
```
Tensor current has NaN values
tensor([[nan, nan, nan,  ..., nan, nan, nan],
        [nan, nan, nan,  ..., nan, nan, nan],
        [nan, nan, nan,  ..., nan, nan, nan],
        ...,
        [nan, nan, nan,  ..., nan, nan, nan],
        [nan, nan, nan,  ..., nan, nan, nan],
        [nan, nan, nan,  ..., nan, nan, nan]], grad_fn=<AddmmBackward0>)
```

**Comment by jpsember** [comment]
```
    if JG.HARD_CODED_NETWORK:
      verify_not_nan("js_model_forward", x)
      x = self.pool1(F.relu(self.conv1(x)))
      x = self.pool2(F.relu(self.conv2(x)))
      x = self.pool3(F.relu(self.conv3(x)))
      x = self.pool4(F.relu(self.conv4(x)))
      x = self.pool5(F.relu(self.conv5(x)))
      x = F.relu(self.conv6(x))
      x = F.relu(self.conv7(x))
      x = F.relu(self.conv8(x))
      x = F.relu(self.conv9(x))
      x = F.relu(self.conv10(x))
      x = F.relu(self.conv11(x))
      x = F.relu(self.conv12(x))
      x = torch.flatten(x, 1)  # flatten all dimensions except batch
      x = F.relu(self.fc1(x))
      verify_not_nan("js_model_forward_final_relu", x)               <---------------------------------
       x = self.fc2(x)
      return x
```
